pymc/tests/test_sampling.py

```diff
@@ -576,13 +576,13 @@ def test_normal_scalar(self):
             assert len(ppc) == 0
 
             # test keep_size parameter
-            ppc = pm.sample_posterior_predictive(trace, keep_size=True, return_inferencedata=False)
+            ppc = pm.sample_posterior_predictive(trace, return_inferencedata=False)
             assert ppc["a"].shape == (nchains, ndraws)
 
             # test default case
             ppc = pm.sample_posterior_predictive(trace, var_names=["a"], return_inferencedata=False)
             assert "a" in ppc
-            assert ppc["a"].shape == (nchains * ndraws,)
+            assert ppc["a"].shape == (nchains, ndraws)
             # mu's standard deviation may have changed thanks to a's observed
             _, pval = stats.kstest(ppc["a"] - trace["mu"], stats.norm(loc=0, scale=1).cdf)
             assert pval > 0.001
```

Comment:
I think it was this one.  If I remember correctly, the p-value was coming out to be something like .0015 or something like that.  I had suggested that it might come down to the random seed and/or the arbitrary threshold of .001, but that my speculation shouldn't be taken as definitive.